src/language-html/utils.js

```diff
@@ -496,7 +496,10 @@ function getNodeCssStyleDisplay(node, options) {
     default:
       return (
         (node.type === "element" &&
-          (!node.namespace || isInSvgForeignObject) &&
+          (!node.namespace ||
+            isInSvgForeignObject ||
+            (!node.hasExplicitNamespace &&
+              !["html", "svg"].includes(node.namespace))) &&
```

Comment:
The `getNodeCssStyleWhiteSpace` function below contains suspiciously similar logic. Could you investigate whether it needs fixing too and whether they can be unified?

Reply:
Right, `<pre>` lost whitespace

Reply:
Working on 3 repos, just now. no time to reply, sorry